src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -883,6 +883,11 @@ public boolean openPairingWindowWithPINCallback(
         deviceControllerPtr, devicePtr, duration, iteration, discriminator, setupPinCode, callback);
   }
 
+  public boolean downloadLogFromNode(
```

Comment:
could you add the doxygen for this public API? thanks

Reply:
Added. Thanks.